diff/parse.go

```diff
@@ -365,7 +365,6 @@ func (r *HunksReader) ReadHunk() (*Hunk, error) {
 		if r.hunk == nil {
 			// Check for presence of hunk header.
 			if !bytes.HasPrefix(line, hunkPrefix) {
-				panic("X")
```

Comment:
This looks like a debugging artifact that was unintentionally left in from commit 86544b82264c7c9e3433dddae90c31fc2a30f5ba.